### PR TITLE
Added frame-size-mb verification function

### PR DESCRIPTION
**Pull request by robertdale**
Fixes issue #199

Signed-off-by: Robert Dale <robdale@gmail.com>

Thank you for contributing to JanusGraph.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [x] Is there an issue associated with this PR? Is it referenced in the commit message?
- [x] Does your PR body contain #xyz where xyz is the issue number you are trying to resolve?
- [x] Has your PR been rebased against the latest commit within the target branch (typically `master`)?
- [x] Is your initial contribution a single, squashed commit?

### For code changes:
- [x] Have you written and/or updated unit tests to verify your changes?
- [x] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)? 
- [x] If applicable, have you updated the LICENSE.txt file, including the main LICENSE.txt file in the root of this repository?
- [x] If applicable, have you updated the NOTICE.txt file, including the main NOTICE.txt file found in the root of this repository?

### For documentation related changes:
- [x] Have you ensured that format looks appropriate for the output in which it is rendered?

### Note:
Please ensure that once the PR is submitted, you check Travis CI for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

**Comment by robertdale** [comment]
@sjudeng Pushed a test.

File: janusgraph-cassandra/src/test/java/org/janusgraph/diskstorage/cassandra/thrift/CassandraThriftStoreManagerTest.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
+package org.janusgraph.diskstorage.cassandra.thrift;
```
Comment (sjudeng) [inline janusgraph-cassandra/src/test/java/org/janusgraph/diskstorage/cassandra/thrift/CassandraThriftStoreManagerTest.java]: Add copyright header

Comment (robertdale) [reply janusgraph-cassandra/src/test/java/org/janusgraph/diskstorage/cassandra/thrift/CassandraThriftStoreManagerTest.java]: Header added.

File: janusgraph-cassandra/src/test/java/org/janusgraph/diskstorage/cassandra/thrift/CassandraThriftStoreManagerTest.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+import org.janusgraph.CassandraStorageSetup;
+import org.janusgraph.diskstorage.configuration.ModifiableConfiguration;
+import org.junit.Test;
+
+public class CassandraThriftStoreManagerTest {
+
+    @Test
+    public void shouldValidateConfigOptionFrameSizeMb() {
+
+        try {
```
Comment (mbrukman) [inline janusgraph-cassandra/src/test/java/org/janusgraph/diskstorage/cassandra/thrift/CassandraThriftStoreManagerTest.java]: Please separate each block into a separate test method, and either use the `@Test(expected = MyRuntimeException.class)` annotation or the `ExpectedException` rule.

For more info, please see:

* https://blog.goyello.com/2015/10/01/different-ways-of-testing-exceptions-in-java-and-junit/
* http://junit.org/junit4/javadoc/4.12/org/junit/rules/ExpectedException.html

The resulting code will be much shorter and easier to read.

Comment (robertdale) [reply janusgraph-cassandra/src/test/java/org/janusgraph/diskstorage/cassandra/thrift/CassandraThriftStoreManagerTest.java]: Done.

File: janusgraph-cassandra/src/test/java/org/janusgraph/diskstorage/cassandra/thrift/CassandraThriftStoreManagerTest.java [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        ModifiableConfiguration config = CassandraStorageSetup.getCassandraThriftConfiguration("janusgraph");
+        config.set(CassandraThriftStoreManager.THRIFT_FRAME_SIZE_MB, 0);
+        config.get(CassandraThriftStoreManager.THRIFT_FRAME_SIZE_MB);
+    }
+
+    @Test
+    public void configOptionFrameSizeMbShouldBeHappy() {
+        ModifiableConfiguration config = CassandraStorageSetup.getCassandraThriftConfiguration("janusgraph");
+        config.set(CassandraThriftStoreManager.THRIFT_FRAME_SIZE_MB, 1);
+        config.get(CassandraThriftStoreManager.THRIFT_FRAME_SIZE_MB);
```
Comment (mbrukman) [inline janusgraph-cassandra/src/test/java/org/janusgraph/diskstorage/cassandra/thrift/CassandraThriftStoreManagerTest.java]: Should this verify that the returned value is 1, not just that it's not crashing?

Comment (sjudeng) [reply janusgraph-cassandra/src/test/java/org/janusgraph/diskstorage/cassandra/thrift/CassandraThriftStoreManagerTest.java]: @mbrukman Sorry, it looked like your earlier feedback was addressed and I merged this too quick.

Comment (mbrukman) [reply janusgraph-cassandra/src/test/java/org/janusgraph/diskstorage/cassandra/thrift/CassandraThriftStoreManagerTest.java]: No worries, it's all good. The beauty of software is that it's always easier to change it (and then change it again) compared with hardware. :-)

Comment (mbrukman) [reply janusgraph-cassandra/src/test/java/org/janusgraph/diskstorage/cassandra/thrift/CassandraThriftStoreManagerTest.java]: This was fixed by @robertdale in PR #560.

**Pull request merged**